### DISCUSSION TITLE
Show documents in "Recent" based on time range

### DISCUSSION THREAD

**Discussion by rYR79435** [jonaswinkler/paperless-ng#937] [Feature Requests]
On the Dashboard in the Recent box, it seems that the last 10 documents are shown, no matter how old they are.
This means that I currently see several documents that are a few months old, which goes against my understanding of "recent".

I would like an option to set a time range, which defines what documents are shown.
E.g. one month, to only show documents no older than 30 days.

Leaving a limit of 10 documents would probably be a good idea, so the Recent box won't ever take up too much space.
(E.g. showing at most 10 documents, no older than 30 days)

**Comment by jonaswinkler** [comment]
Well, your "Recent" view shows documents with whatever filtering and sorting settings you applied when you saved the "Recent saved view.

The standard date filters don't really support relative dates. Here's a neat little trick you can do with the full text search / advanced search: Search for `added:'-30d to now'`. That does not support sorting (yet), but should be pretty close.

**Reply by rYR79435** [reply]
Wow, so I created that view myself? I cannot remember that at all!

Thank you for the advanced search tip, that works for me.
